.github/workflows/pyvista.yml

```diff
@@ -2,18 +2,18 @@ name: Test visualisation demos
 
 on:
   # Uncomment the below to trigger tests on push
-  # push:
-  #   branches:
-  #     - "**"
+  push:
+    branches:
+      - "**"
```

Comment:
Should this be on for every push or just for PRs to main?

Reply:
It is just to be able to check that the CI passes, once it is green I'll revert it